lib/awful/spawn.lua

```diff
@@ -323,14 +323,12 @@ end
 --- Asynchronously spawn a program and capture its output.
 -- (wraps `spawn.with_line_callback`).
 -- @tparam string|table cmd The command.
--- @function callback Function with the following arguments:
+-- @tab callback Function with the following arguments
```

Comment:
I see, agreed.
From 289bc26a (where I've added it) I was under the impression that it even should be there by default.